dronebl/__init__.py

```diff
@@ -38,11 +38,12 @@ def _post(self,
         return response.read()
 
     def lookup(self,
-            query: Union[str, int],
-            type:  Optional[int]=None,
-            limit: Optional[int]=None
+            query:  Union[str, int],
+            type:   Optional[int]=None,
+            limit:  Optional[int]=None,
+            listed: Optional[int]=2
```

Comment:
where did 2 come from? is that in a doc somewhere?

Reply:
listed="0|1|2" - from https://dronebl.org/docs/rpc2

0 is not listed, 1 is listed, so 2 must be any (it is also the default if the param is not given)

Reply:
can you test this just to be sure?

Reply:
I have